modules/promotion/src/Element/CouponRedemptionForm.php

```diff
@@ -32,6 +32,9 @@ public function getInfo() {
     return [
       // The order to which the coupon will be applied to.
       '#order' => NULL,
+      '#code_label' => t('Coupon code'),
+      '#code_description' => t('Enter your coupon code here.'),
+      '#apply_label' => t('Apply coupon'),
```

Comment:
I think people would expect the coupon element to inherit the main title and description. Which leaves only the apply button as needing a custom title (#apply_title or #submit_title I guess)